Core/Src/compute.c

```diff
@@ -92,9 +92,76 @@ uint8_t compute_init(acc_data_t *bmsdata)
 
 	HAL_ADC_Start(&hadc2);
 
+	/* Initializing rate limited messages */
+	init_rl_config();
+
 	return 0;
 }
 
+void init_rl_config()
+{
+	can_msg_t discharge_msg = { 0 };
+	discharge_msg.id =
+		DISCHARGE_CANID; // 0x0A is the dcl id, 0x22 is the device id set by us
+
+	can_msg_t charge_msg = { 0 };
+	charge_msg.id =
+		CHARGE_CANID; // 0x0A is the dcl id, 0x157 is the device id set by us
+
+	can_msg_t acc_status_msg;
```

Comment:
You should put message lengths in the config since they do not change and are properties of the message.